azure-pipelines.yml

```diff
@@ -1,162 +1,888 @@
-
-      - task: CmdLine@2
-        displayName: 'Build Test App Local'
-        inputs:
-          script: |
-            pnpm build-test-app-local
-          workingDirectory: '$(ClientSdkProjectDirectory)'
-
-      - bash: 'node tools/cli/runAppsWithE2ETests.js --reportFileName=e2e-tests-report-local-script-tag --envType=localScriptTag'
-        displayName: 'Run E2E integration tests with local script tag'
-        condition: succeeded()
-        workingDirectory: '$(AppHostingSdkProjectDirectory)'
-
-      - task: PublishTestResults@2
-        inputs:
-          testResultsFormat: 'JUnit'
-          testResultsFiles: '**/e2e-tests-report*.xml'
-          testRunTitle: 'E2E Tests - Local Tag'
-          mergeTestResults: true
-        condition: succeededOrFailed()
-
-  - job: E2ETestCDN
-    displayName: 'E2E Test - CDN (only runs on release builds)'
-    # This test only runs after deployment from a release branch and the new CDN version has been deployed
-    # This check will run on the PR to merge the release branch back into main
-    condition: and(
-      eq(variables['Build.Reason'], 'PullRequest'),
-      startsWith(variables['System.PullRequest.SourceBranch'], 'release/'),
-      eq(variables['System.PullRequest.TargetBranch'], 'main')
-      )
-    pool:
-      vmImage: 'ubuntu-latest'
-    steps:
-      - template: tools/yaml-templates/build-app-host.yml
-        parameters:
-          appHostGitPath: git://$(AppHostingSdkGitPath6)@$(AppHostingSdkGitRef)
-
-      - task: CmdLine@2
-        displayName: 'Build Test App CDN'
-        inputs:
-          script: |
-            pnpm build-test-app-CDN
-          workingDirectory: '$(ClientSdkProjectDirectory)'
-
-      - bash: 'node tools/cli/runAppsWithE2ETests.js --reportFileName=e2e-tests-report-cdn-script-tag --envType=cdnScriptTag'
-        displayName: 'Run E2E integration tests with local script tag on latest cdn bundles'
-        condition: succeeded()
-        workingDirectory: '$(AppHostingSdkProjectDirectory)'
-        enabled: true
-
-      - task: PublishTestResults@2
-        inputs:
-          testResultsFormat: 'JUnit'
-          testResultsFiles: '**/e2e-tests-report*.xml'
-          testRunTitle: 'E2E Tests - CDN'
-          mergeTestResults: true
-        condition: succeededOrFailed()
-
-  - job: E2ETestAndroid
-    displayName: 'E2E Test - Android'
-    pool:
-      vmImage: 'internal-macos-11'
-    steps:
-      - template: tools/yaml-templates/android-test.yml
-        parameters:
-          androidAppHostingSdkGitPath: git://$(AndroidAppHostingSdkGitPath)
-
-  - job: E2ETestIOS
-    displayName: 'E2E Test - IOS'
-    pool:
-      vmImage: 'internal-macos12'
-    steps:
-      - template: tools/yaml-templates/ios-test.yml
-        parameters:
-          iOSAppHostingSdkGitPath: git://$(IOSAppHostingSdkGitPath)
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-2022
+      os: windows
+    sdl:
```

Comment:
We are supposed to use this if we aren't using windows-latest as our pipeline. We are using it...sometimes. I don't know 100% if this is required or not.